config-overrides.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function override(webpackConfig) {
```

Comment:
what is this file doing ?

Reply:
One new module uses cjs files that aren't supported by create-react-app for some reason.
This fix is based on this issue:
https://github.com/ealmansi/set-interval-async/issues/49

Reply:
Maybe you should add a comment with that explanation ?